gatsby-node.js

```diff
@@ -300,12 +301,14 @@ const localFileCheck = async (
       // `data` is direct from Airtable so we don't use
       // the cleanKey here
       data[key].forEach(attachment => {
+        const ext = mime.getExtension(attachment.type) // unknown type returns null
         let attachmentNode = createRemoteFileNode({
           url: attachment.url,
           store,
           cache,
           createNode,
-          createNodeId
+          createNodeId,
+          ext: ext ? `.${ext}` : undefined
```

Comment:
```suggestion
          ext: !!ext ? `.${ext}` : undefined
```
I tend to prefer a double bang (`!!`) to be certain we have a boolean and avoid accidentally truthy/falsey errors. It may not be an issue with `mime`, but not worth the presumption when you can be certain.

Reply:
yep, makes sense. I just committed it 👌